05.ls/ls.rb

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require 'etc'
+
+options = ARGV.getopts('a', 'l', 'r')
+
+array = []
+
+if options['l'] == true
```

Comment:
Ruby だと特に強い理由がなければ `true` と比較しないことが多いです。
なので問題ないようでしたら `== true` を取っちゃった方がそれっぽいかもしれませんね。

```suggestion
if options['l']
```

Reply:
ありがとうございます。修正しました。